api/capi/src/nntrainer.cpp

```diff
@@ -83,7 +84,7 @@ int ml_nnmodel_construct_with_conf(const char *model_conf, ml_nnmodel_h *model)
 
   std::ifstream conf_file(model_conf);
   if (!conf_file.good()) {
-    std::cerr << "Error: Cannot open model configuration file : " << model_conf << std::endl;
+    ml_loge("Error: Cannot open model configuration file : %s", model_conf);
```

Comment:
Q. is `model_conf` always a valid pointer? (because conf_file.good() was failed)

Reply:
conf_file.good() will check the validation of model_conf.

Reply:
I meant that if `model_conf` is nullptr, this log message may cause segmentation fault.
So, I wonder that the null check of model_conf was already done in a caller.